apichangelog.md

```diff
@@ -1,6 +1,7 @@
 ## API Change Logs
 This document provides information about new features and bug fixes with the Lightroom APIs.
 
-### Version 1.1.0 (21 Dec, 2020)
-- Added new get `catalogs/assets/{asset_id}`. Api that retrieves information about a single asset in a catalog that the caller owns.
-- Added new get `catalogs/albums/{album_id}/assets`. Api that lists assets in an album.
+### Version 1.1.0 (11 Jan, 2021)
```

Comment:
should we keep the older change log instead of removing? maybe place that block after the newly added block.

Reply:
older logs were added just for POC of api change logs, peter also mentioned that api change logs will accommodate only new changes, thats removed earlier one